lib/utils/joinPath.test.ts

```diff
@@ -1,10 +1,6 @@
 import { joinPath } from './joinPath';
 
 describe('joinPath()', () => {
-  test('handles empty path', () => {
-    expect(joinPath([])).toEqual('');
-  });
```

Comment:
note: test case not valid anymore since `joinPath` does NOT accept empty arrays at ts level